src/Traits/ActionTrait.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Opstalent\CrudBundle\Traits;
+
+use Opstalent\CrudBundle\Annotations\AbstractAnnotation;
+
+/**
+ * Trait ActionTrait
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+trait ActionTrait
+{
+    /**
+     * @var string[]
+     * @Required
+     * @AnnotatedDescription("The property holds all available actions for class")
+     */
+    public $actions = [];
+
+    /**
+     * @param string $key
+     * @param array $data
+     * @return array
+     */
+    protected function setActions(array $data): array
```

Comment:
is good to reason for setActions method return result of getActions()? unintuitive and mismatch responsibility in my opinion